TravelBuddies/TravelBuddies/Base.lproj/Main.storyboard

```diff
@@ -47,7 +47,7 @@
                 </viewController>
```

Comment:
On the main storyboard, you can make the view controllers be presented in full screen. Selecting the view, then on the fifth tab on the right (downward pointing arrow), and then in presentation.